packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -62,7 +95,15 @@ const TransferPanel = (): JSX.Element => {
 
   const [l1Amount, setL1AmountState] = useState<string>('')
   const [l2Amount, setL2AmountState] = useState<string>('')
+
   const { shouldDisableDeposit } = useWithdrawOnly()
+
+  useEffect(() => {
+    if (connectionState === ConnectionState.DEPOSIT_MODE) {
```

Comment:
@DZGoldman should this also include `ConnectionState.WITHDRAW_MODE`? I don't think it's possible for the bridge to be in this state initially, without the user switching to it manually?

That being said, in the future iterations of the Bridge, I'd like to move as much state like this (deposit/withdraw, token address) as possible to the search params. It will speed up the user as they wouldn't have to start each session from scratch.

Reply:
Yep, exactly — currently the page always loads in  deposit mode, but we may as well allow deposit and withdrawal mode in this conditional for future-proofing.

Reply:
So there's some confusing naming here / overloaded terms that we should probably change; the `AppState.connectionState` property tells which network the wallet is currently connected to; i.e., if it's `ConnectionState.DEPOSIT_MODE` wallet is connected to L1, `ConnectionState.WITHDRAW_MODE`, wallet is connected to L2. So you actually can have an initial state of ConnectionState.WITHDRAW_MODE.

The state that tracks whether the pannel reads "Deposit" or "Withdraw" is `AppState.isDepositMode`. :/ 

Probably we should change the ConnectionState enum options to something like L1_CONNECTED vs L2_CONNECTED.

Reply:
(I'll keep calling it ConnectionState.WITHDRAW_MODE for now): For this conditional, we'll want to show the modal on WITHDRAW_MODE or DEPOSIT_MODE, but only on the initial load; i.e., with no other changes, it will redisplay the modal whenever the network switches.

Reply:
(we can do the renaming I mentioned above in a separate PR after this one for diff cleanliness if that's nicer)